apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -824,29 +873,42 @@ repack(Start, End, NextCursor, RightBound, RequiredPacking, StoreID) ->
 						ok;
 					(AbsoluteOffset, {_, TXRoot, _, _, _, ChunkSize}, ok) ->
 						PaddedOffset = ar_data_sync:get_chunk_padded_offset(AbsoluteOffset),
-						case ar_sync_record:is_recorded(PaddedOffset, ar_data_sync) of
-							{{true, RequiredPacking}, StoreID} ->
+						case ar_sync_record:is_recorded(PaddedOffset, ar_data_sync, StoreID) of
+							{true, RequiredPacking} ->
 								?LOG_WARNING([{event,
```

Comment:
This shouldn't happen right? Since the cursor and get_next_synced_interval should prevent us from trying to repack a chunk that has already been repacked

Reply:
The cursor is not strictly in sync with the process and serves mostly to save time after a restart. However, since we are only looking at the intervals packed with a different packing this here should not happen, yes.

Reply:
:+1: thanks.